### DISCUSSION TITLE
Bandwidth consumed by Rport and how to reduce it

### DISCUSSION THREAD

**Discussion by oneline99** [openrport/openrport#61] [General]
HI , does anyone here know how rport client manages its keep alive communication with server ? I noticed that it is transmitting about 1.5 Mb every hour without monitoring or updates. I would like to reduce this bandwidth consumed. Is there a way to do so ?

**Comment by alhdo** [comment]
Hello, 
       You can play with the keepalive options. It use ssh to communicate with the server
